ui/app/lib/console-helpers.js

```diff
@@ -74,7 +71,17 @@ export function parseCommand(command, shouldThrow) {
       flags.push(arg);
     } else {
       if (path) {
-        data.push(arg);
+        let strippedArg = arg
+          // we'll have arg=something or arg="lol I need spaces", so need to split on the first =
+          .split(/=(.+)/)
+          // remove " at the beginning and end of each item
+          .map(item => item.replace(/^"|"$/gi, ''))
```

Comment:
🤔 maybe we should strip single quotes here too...

Reply:
updated this so it 1) also strips single quotes, and 2) will only do so if the quotes are matching